src/style.css

```diff
@@ -0,0 +1,126 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(/fonts/Avenir-Book.ttf);
+
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(/fonts/Avenir-Heavy.ttf);
+
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-icon {
+  display: inline-block;
+  position: relative;
+}
+
+.search-box {
+  position: relative;
+}
+
+.search-box:first-child {
+  margin-top: 20px;
+}
+
+.big-search-icon {
+  position: absolute;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+  bottom: 26px;
+}
+
+.small-search-icon {
+  position: absolute;
+  width: 11px;
+  top: 15px;
+  bottom: 16px;
+  left: 13px;
+}
+
+.search-text {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.big-search {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small-search {
+  width: 100%;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 33px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.big-search:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.big-search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
your search bars have an outline on focus 🥲

Reply:
Hello. I am not sure I've understood the mistake here. I've copied background and other settings from Figma. Have I missed something, or added extra?

Reply:
you did not add some rule, to remove the blue outline 😉
there is no outline in the design, but you have it 😁
<img width="226" alt="image" src="https://user-images.githubusercontent.com/55802630/172351033-51eea48f-6b58-4046-a399-cb7ac3089086.png">